pkg/grpc/universalapi/api_subtlecrypto_subtlecrypto.go

```diff
@@ -28,13 +28,21 @@ import (
 	"github.com/lestrrat-go/jwx/v2/jwk"
 
 	contribCrypto "github.com/dapr/components-contrib/crypto"
+	"github.com/dapr/dapr/pkg/buildinfo"
 	diag "github.com/dapr/dapr/pkg/diagnostics"
 	"github.com/dapr/dapr/pkg/messages"
 	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	"github.com/dapr/dapr/pkg/resiliency"
 	kitCrypto "github.com/dapr/kit/crypto"
 )
 
+// TODO: Remove this when the build tag is removed
+func init() {
+	// Add a fake feature "_SubtleCrypto" which is not a real preview feature
+	// This is then exposed in the sidecar's metadata endpoint so tests (and users) can query for this.
+	buildinfo.AddFeature("_SubtleCrypto")
```

Comment:
Exposing `_SubtleCrypto` as a feature flag makes it visible through the metadata APIs (thanks to #6537 ), so the E2E tests can decide if they should skip the subtle crypto tests